src/app/personside/visittkort/body/familie/Foreldre.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import Undertekst from 'nav-frontend-typografi/lib/undertekst';
+
+import VisittkortElement from '../VisittkortElement';
+
+import { Familierelasjon, getMorOgFar, Kjønn, Relasjonstype } from '../../../../../models/person';
+import BorMedBruker from '../../../../../components/person/HarSammeBosted';
+import NavnOgAlder from '../../../../../components/person/NavnOgAlder';
+import { utledKjønnFraFødselsnummer } from '../../../../../utils/fnr-utils';
+
+const mannPath = require('../../../../../resources/svg/mann.svg');
+const kvinnePath = require('../../../../../resources/svg/kvinne.svg');
+
+interface Props {
+    familierelasjoner: Familierelasjon[];
+}
+
+interface ForelderProps {
+    relasjon: Familierelasjon;
+}
+
+export function Forelder({relasjon}: ForelderProps) {
+    const beskrivelse = relasjon.rolle === Relasjonstype.Mor ? 'Mor' : 'Far';
+    const kjønn = utledKjønnFraFødselsnummer(relasjon.tilPerson.fødselsnummer);
```

Comment:
tror ikke vi kan gjøre dette ref praten på demo. Vi må bruke felt fra TPS. 
Det har vi vel sikkert ikke tilgjengelig uten å måtte gjøre et nytt TPS-oppslag så da stemmer jeg nesten bare for at det står Enke uansett (det som TPS leverer)

Reply:
Ser den. Har lagt ut spørsmål til freg på slack. Hva tror du om at vi lager en issue, merger denne så går vi heller tilbake og fikser det?

Reply:
Hva tror du @janeirik ?

Reply:
La inn en kommentar på OPP-396.